data/korath/korath missions.txt

```diff
@@ -257,3 +257,223 @@ mission "Small Scale Delivery"
+			`	You return to the Korath ship. In the living quarters you carefully gather all the personal effects you can find and box them for transport. You've no way of knowing if those in the photos are now safely recovering or lying in your hold.`
+			`	The crew makes a last sweep to move debris, both to assess for repairs later and to make sure you haven't missed anyone, or parts of them. You clean and disinfect as best you can and return to your ship for the trip to <destination>.`
+
+	on complete
+		conversation
+			`A crowd pours out of the spaceport when you land. It takes some time for the workers to establish barricades and clear a path so the transports can get to your cargo hold. A team in uniform greets you formally when you open the hatch. You count the coffins together and show them the boxes of possessions.`
+			`	The officer thanks you and hands you a data chip that vaguely reminds you of a license.`
+			`	The crowd has been somber, but when the first coffin leaves your ship, they erupt in a cachophony of barking sounds. Some Korath try to rush forward but are held back by others, you presume their family.`
+			`	You go back inside your ship rather than intrude on their grief. You slot the data chip into your systems but the information is in Korathi. You set it aside and work on a salvage plan.`
+			`	As you review your notes on the ship and the number of crew they were carrying, it's clear that you will need to carry at least 66 additional crew that you can communicate with to be able to salvage the <EDC-ship>. The closest system where you can find that many Human speakers is on Greeenwater in Hai space.`
+
+mission "Efreti: Distress Call 3"
+	name "Pick up salvage crew"
+	description "Pick up salvage crew."
+	source "Karek Fornati"
+	destination "Greenwater"
+	landing
+	
+	to offer
+		has "Efreti: Distress Call 2: done"
+	
+	on complete
+		conversation
+			`For all the stories about how the Hai like change and excitement, not many are interested in travelling into the territory of a species that raided their worlds in living memory to salvage a ship that has already been caught in the crossfire of the Mereti and Sestor war.`
+			`	Fortunately, there are plenty of Humans who are willing to risk it for the right pay, even given the comfortable life they have in Hai space. You show the new crew members to their bunks and prepare to salvage the <EDC-ship>.`
+
+ship "Charm-Shallop" "Charm-Shallop (Distressed)"
+	outfits
+		"Warder Anti-Missile"
+		"Banisher Grav-Turret"
+		
+		"Generator (Candle Class)"
+		"Large Heat Shunt"
+		"Fuel Processor"
+		"Thermal Repeater Rifle" 66
+		
+		"Gaktem GP Hybrid Thruster"
+		"Gaktem GP Hybrid Steering"
+		"Scram Drive"
+
+mission "Efreti: Distress Call 4"
+	name "Salvage ship"
+	description "Salvage the Korath ship."
+	source "Greenwater"
+	stopover "Seleptra Nak"
+	destination "Karek Fornati"
+	landing
+	passengers 66
```

Comment:
Given that this uses a stopover, I will still have these 66 passengers after collecting the ship.
I'm fairly sure the ship will come fully crewed when it is given, but I have not checked.